### PR TITLE
ci(): migrate scripts to es6 modules

### PR DESCRIPTION
**Pull request by ShaMan123**
https://nodejs.org/api/esm.html#enabling

migrate scripts to es6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,17 +51,17 @@
   },
   "license": "MIT",
   "scripts": {
+    "fabric": "node ./scripts/index.mjs",
```
Comment (asturur) [inline package.json:54]: can we call this CLI or something else that is not fabric? sounds confusing to me.

File: scripts/index.mjs [hunk 2]
```diff
@@ -13,25 +13,31 @@
...
+import inquirer from 'inquirer';
+import Checkbox from 'inquirer-checkbox-plus-prompt';
+import killPort from 'kill-port';
+import _ from 'lodash';
+import moment from 'moment';
+import path from 'node:path';
+import process from 'node:process';
+import { fileURLToPath } from 'node:url';
+import os from 'os';
+import { listFiles, transform as transformFiles } from './transform_files.mjs';
```
Comment (asturur) [inline scripts/index.mjs:30]: can MJS import normal commonJS modules if needed?

File: package.json (hunk 1 above)
Comment (ShaMan123) [reply package.json]: done

File: scripts/index.mjs (hunk 2 above)
Comment (ShaMan123) [reply scripts/index.mjs:30]: seems possible
there's also an option to import require and use that, see [SO](https://stackoverflow.com/a/56350495/9068029)

https://nodejs.org/api/esm.html#differences-between-es-modules-and-commonjs

**Review by asturur — APPROVED** [review]

**Pull request merged**